utils/xyk.ts

```diff
@@ -57,7 +58,7 @@ export class Xyk {
     soldAssetId: BN,
     boughtAssetId: BN,
     boughtAssetAmount: BN,
-    maxAmountIn: BN = new BN(Number.MAX_SAFE_INTEGER)
+    maxAmountIn: BN = new BN("340282366920938463463374607431768211455") //u128::MAX
```

Comment:
what is the difference? :O

Reply:
max safe int:                                                                  9_007_199_254_740_991
u128::MAX.    340_282_366_920_938_463_463_374_607_431_768_211_455
the first one is less then 1MG with 18 decimals, not usable for our `Balance: u128` type